charts/agent/README.md

```diff
@@ -169,11 +169,11 @@ Chart to install K8s collection stack based on Observe Agent
 | cluster.metrics.pod.action | string | `"keep"` |  |
```

Comment:
is this used anywhere then if we're defaulting to keep?

Reply:
that option is gone now - seeing as we have enable as a value it is set to keep all the time in config and it just won't be added if we don't set enabled = true